src/App.js

```diff
@@ -999,6 +1012,35 @@ class App extends Component {
                 });
               }
 
+              if (!userVotes){
+                const userVotes = {};
+                const localTree = getStoredValue("votes", account);
+                const parsedTree = JSON.parse(localTree);
+
+                if (parsedTree){
+                  for (const key in parsedTree){
```

Comment:
can be replaced with `for (const [key, value] of parsedTree)` which checks for `hasOwnProperty` itself

Reply:
ignore me :)